### ISSUE TITLE
[Suggestion] It would be better to remove the containerized function app when executing containerized function app unsupported commands

### ISSUE THREAD

**Issue by v-ruizh** [microsoft/vscode-azurefunctions#4174]
**OS:** All
**Build Version:** v1.15.1-alpha.0
**Regression:** Not a Regression

**Repro Steps:**
1. Create a containerized function app.
2. F1 -> Select "Azure Functions: Configure Deployment Source...".
3. Check whether containerized function app shows in the function app list.

**Actual:**
The containerized function app shows in the function app list.
![image](https://github.com/microsoft/vscode-azurefunctions/assets/45614674/6461e96e-3cb2-4ac2-8302-8514f61c7c73)

**Suggestion:**
It would be better to remove the containerized function app when executing containerized function app unsupported commands.

**More Info:**
1. Fail to execute "Configure Deployment Source..." command for a containerized function app.
![image](https://github.com/microsoft/vscode-azurefunctions/assets/45614674/b962c7c6-b8bd-4f50-b157-4b0dc637498c)
2. This bug also reproduces when executing other containerized function app unsupported commands, such as "Start/Stop Streaming Logs", "Start/Restart/Stop", "Deploy to Slot"...

**Comment by motm32** [comment]
Hello! Thanks for filing this issue. Even though the containerized function app does show in the quick pick when a containerized function app is chosen for these unsupported commands an error like this should show up: 
![image](https://github.com/microsoft/vscode-azurefunctions/assets/59709511/55e8e7bb-da35-4759-be12-85b31d829358)
Please let me know if that is not the case!

**Comment by v-ruizh** [comment]
Yes, I added the more info in the bug. There is an error when executing the unsupported commands for containerized function app.  I saw you have fixed the same issue for Deploy commands, so log a new bug for other commands. 

Is this by design for other unsupported commands?
